### ISSUE TITLE
mobile devices?

### ISSUE THREAD

**Issue by leftstick** [rook2pawn/node-ddos#12]
So far as i know, multiple mobile devices may share the same public IP address, which might gets blocked by your ddos module, is it possible handling this scenario?

Thanks

**Comment by rook2pawn** [comment]
@leftstick Yes, the user agent string handles this scenario as the host identification is a combination of IP and user-agent. However if both mobile devices are using the same user-agent browser, and on the same IP, then it is best to whitelist, which I will be addressing in another commit that is also addressed in another issue #10 

Whitelist issue addressed and commited https://github.com/rook2pawn/node-ddos/commit/2640c00a03ac8378b71c135d49bf136db871dd54

**Comment by leftstick** [comment]
Thanks @root2pawn
